nodeEditor/src/index.ts

```diff
@@ -1 +1,2 @@
-export * from "./nodeEditor";
\ No newline at end of file
+export * from "./nodeEditor";
+export * from "./sourceEditor/shaderSourceEditor";
```

Comment:
This should not be here (I would like to avoid having dependencies between both editors)

Reply:
You mean move it to project root? I just put it in there to save you the trouble of maintaining a separate npm module for it, currently it would share `babylonjs-node-editor`. It can be moved to project root and separated from the NME if desired.

Reply:
No I meant, just move it into the inspector. we have other editor there. THe only one which is separated is NME because it is independant. No need to have a module for the Shader Material Editor